README.md

```diff
@@ -466,6 +468,25 @@ block-beta
 ```bash
 protoflow execute ReadFile path=/tmp/file.txt
 ```
+#### [`ReadSocket`]
```

Comment:
Leave an empty line prior to the section header

Reply:
done